docker/j9.Dockerfile

```diff
@@ -1,7 +1,8 @@
 ARG git_ref
 FROM ghcr.io/projectkig/kigpaper/kig-paper-builder:${git_ref} AS builder
 
-FROM ibmjava:8-jre-alpine
+# waiting for an official ibm-semeru-runtimes alpine image
+FROM cobaltinc:openj9-17.0.3_7-jre-alpine
```

Comment:
Which container is this pulling? I found a container `cobaltinc/openj9-17.0.3_7-jre-alpine`, but the tag for that is 
the default `latest`.

Seeing as this container doesn't use tags and doesn't provide any information or source, I'm not comfortable using it in a production environment or releasing a Dockerfile that blindly trusts this image.
Are we going to use Hotspot until OpenJ9 on Alpine is released somewhere? If so, I believe we can just leave this Dockerfile broken for now and update once we have a reliable source. Alternatively, we could use a non-Alpine-based image, as we're not planning to use this anyway at the moment. This has the added benefit of not breaking our CI.

Reply:
I agree, we can keep the J9 image on JRE 8 for now.

Ideally, we'd also want to use the Hotspot image just for building. We should build our own JRE image with `jlink`, but that should target the whole infrastructure, so it's not something for this repository.